nautobot/core/views/generic.py

```diff
@@ -302,28 +301,30 @@ def get(self, request):
         table_config_form = None
         current_saved_view = None
         current_saved_view_pk = self.request.GET.get("saved_view", None)
-        list_url = validated_viewname(model, "list")
         # We are not using .restrict(request.user, "view") here
         # User should be able to see any saved view that he has the list view access to.
         if user.has_perms(["extras.view_savedview"]):
-            saved_views = SavedView.objects.filter(view=list_url).order_by("name").only("pk", "name")
+            saved_views = SavedView.objects.filter(view=view_name).order_by("name").only("pk", "name")
         else:
             shared_saved_views = (
-                SavedView.objects.filter(view=list_url, is_shared=True).order_by("name").only("pk", "name")
+                SavedView.objects.filter(view=view_name, is_shared=True).order_by("name").only("pk", "name")
             )
             user_owned_saved_views = (
-                SavedView.objects.filter(view=list_url, owner=user).order_by("name").only("pk", "name")
+                SavedView.objects.filter(view=view_name, owner=user).order_by("name").only("pk", "name")
             )
             saved_views = shared_saved_views | user_owned_saved_views
+
+        # Construct the objects table
```

Comment:
Comment seems out of place - shouldn't it be inside the `if self.table` block still?